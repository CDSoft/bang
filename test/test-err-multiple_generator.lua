generator(false)

generator {} -- [test/test-err-multiple_generator.lua:3] ERROR: generator: multiple call
