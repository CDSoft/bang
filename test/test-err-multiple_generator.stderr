[test/test-err-multiple_generator.lua:3] ERROR: generator: multiple call
